README.md

```diff
@@ -19,6 +19,9 @@ Required variables:
 - `idr_instance_network_networks`: A list of networks the server should be connected to.
   If the instance is connected to other networks which aren't in this list they *won't* be removed from the server.
 
+Optional variables:
+- `idr_instance_networks_server_key`: Whether to lookup servers by name or id, default `name`
```

Comment:
Why does `networks` here have the "s"? Ok to use the same prefix as the other variables, e.g. `idr_instance_network_`?